src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -16,7 +16,10 @@
 from .access_control import LocalAuthenticator, UserDoesNotExist
 
 cherrypy.config.update({
-    'response.headers.server': 'Ceph-Dashboard'
+    'response.headers.server': 'Ceph-Dashboard',
+    'response.headers.content-security-policy': "frame-ancestors 'self';",
```

Comment:
Sorry for being late, but has someone tested if this setting works in conjunction with [GRAFANA_FRONTEND_API_URL](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/settings.py#L79)?

Reply:
Nope, I didn't test with this setting. I'm not sure how to test it , maybe you or @p-se can test and verify it's working?

Reply:
Doesn't affect `GRAFANA_FRONTEND_API_URL`.

Reply:
Thanks @p-se for verifying it.